doc/usage/configuration.rst

```diff
@@ -660,10 +660,11 @@ documentation on :ref:`intl` for details.
    generated by Sphinx will be in that language.  Also, Sphinx will try to
    substitute individual paragraphs from your documents with the translation
    sets obtained from :confval:`locale_dirs`.  Sphinx will search
-   language-specific figures named by `figure_language_filename` and substitute
-   them for original figures.  In the LaTeX builder, a suitable language will
-   be selected as an option for the *Babel* package.  Default is ``None``,
-   which means that no translation will be done.
+   language-specific figures named by `figure.language.filename` 
+   (e.g. the German version of ``myfigure.png`` will be ``myfigure.de.png``) 
```

Comment:
Note: this filename is generated by the default setting of `figure_language_filename`. So we should add a phrase like "by default setting".